### DISCUSSION TITLE
How many bounding boxes predicts YOLOv5 per grid cell? (B)

### DISCUSSION THREAD

**Discussion by jeannotdamoiseaux** [ultralytics/yolov5#8221] [Q&A] [answered]
I read [here](https://www.v7labs.com/blog/yolo-object-detection) that YOLOv2 predicts 5 bounding boxes per cell, and v3 only 3 (but in 3 different scales, so basically 9). How many bounding boxes predicts v5 per cell?

Also, I was wondering how the algorithm determines the total number of cells for each image?

**Comment by glenn-jocher** [comment, marked as answer]
@jeannot-github default is 3 anchors per cell.

**Comment by jeannotdamoiseaux** [comment]
How is it possible that the model benefits from having multiple predictions per cell? Why does the same cell result in multiple unique predictions?

**Reply by glenn-jocher** [reply]
Multiple predictions per cell with multiple anchors has been the YOLO default for a long time. This is informed from empirical results, like everything else.
